lib/stream.c

```diff
@@ -602,6 +602,17 @@ double stream_getd(struct stream *s)
 	return u.r;
 }
 
+void stream_verify(struct stream *s, size_t size)
+{
+	CHECK_SIZE(s, size);
```

Comment:
Nack on this. Truncates `size` if it won't fit in the stream, and it's deprecated besides.